Casks/a/avogadro.rb

```diff
@@ -1,8 +1,11 @@
 cask "avogadro" do
-  version "1.97.0"
-  sha256 "feb33b01de7f83e74ccae47f251803baeef6de727b3a19622921bfa7d289f631"
+  arch arm: "-arm64", intel: ""
```

Comment:
This can be simplified.

```suggestion
  arch arm: "-arm64"
```